code/modules/atmospherics/machinery/components/unary_devices/outlet_injector.dm

```diff
@@ -68,12 +68,7 @@
 	var/datum/gas_mixture/air_contents = airs[1]
 
 	if(air_contents.temperature > 0)
-		var/transfer_moles = (air_contents.return_pressure() * volume_rate) / (air_contents.temperature * R_IDEAL_GAS_EQUATION)
-
-		if(!transfer_moles)
-			return
-
-		var/datum/gas_mixture/removed = air_contents.remove(transfer_moles)
+		var/datum/gas_mixture/removed = air_contents.remove_ratio(volume / volume_rate)
```

Comment:
volume doesn't exist here, you'll need to do air_contents.volume I'd think. That or use the turf volume define, but that's perhaps a bit weak